### PR TITLE
Instructor ranking

### PR DESCRIPTION
**Pull request by lauradobson**
Includes:

1. Instructor ranking functionality
2. Writes to the db
3. Preloads current db data
4. Each hash is unique per course

### CODE REVIEW

File: taworks/taform/templates/taform/instructor_ranking.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  </tr>
+  {% for i in student %}
+  <tr>
+    <td>{{ i.first_name}}</td>
+    <td>{{ i.last_name }}</td>
+    <td>{{ i.email }}</td>
+    <td>{{ i.reason}}</td>
+    <td>{% if i.cv %}
+   			<a href={{ i.cv.url }} target="_blank">Resume</a>
+		{% else %}
```
Comment (j2kan) [inline taworks/taform/templates/taform/instructor_ranking.html]: line 32-line 35 needs to be line up with the opening <td> and {if}

File: taworks/taform/templates/taform/test.html [hunk 2]
```diff
@@ -1,5 +1,57 @@
 {% load static %}
```
Comment (j2kan) [inline taworks/taform/templates/taform/test.html]: can we rename this file name?

Comment (j2kan) [reply taworks/taform/templates/taform/test.html]: slash instructor_ranking and test.html seem to be for the same page. keep 1 get rid of the other?

File: taworks/taform/urls.py [hunk 3]
```diff
@@ -18,4 +18,6 @@
     url(r'upload_front_matter.html', views.upload_front_matter, name='upload_front_matter'),
     url(r'ranking_status.html', views.ranking_status, name='ranking_status')
 ]
-urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
+
+if settings.DEBUG is True:
```
Comment (j2kan) [inline taworks/taform/urls.py]: get rid of this

File: taworks/taform/views.py [hunk 4]
```diff
@@ -17,6 +17,11 @@
 import mimetypes
 from wsgiref.util import FileWrapper
 from access_tokens import scope, tokens
+import uuid 
+from taform.models import Course, Application, Student
```
Comment (j2kan) [inline taworks/taform/views.py]: choose 1 style either keep the import and change it all

**Review by j2kan — DISMISSED** [review]
just a few comments lgtm

**Review by amyleblond — DISMISSED** [review]
YAYY

**Review by j2kan — APPROVED** [review]
LGTM

**Pull request merged**